### PR TITLE
Update Agent README.md for workflow

### PR DESCRIPTION
**Pull request by louie-tsai**
## Description

Added a workflow for AgentQnA following what ChatQnA has. [ChatQnA flow](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA#architecture-and-deploy-details)

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

NA

## Tests

NA

### CODE REVIEW

File: AgentQnA/README.md [hunk 1]
```diff
@@ -5,6 +5,73 @@
...
+        EM([Embedding MicroService]):::blue
+        RET([Retrieval MicroService]):::blue
+        RER([Rerank MicroService]):::blue
+    end
+    subgraph UserInput[" User Input "]
+        direction LR
+        a([User Input Query]):::orchid
+        Ingest([Ingest data]):::orchid
+    end
+    AG_REACT([Agent MicroService - react]):::blue
```
Comment (xiguiw) [inline AgentQnA/README.md:39]: the workflow looks good.
image
Could the supervisor agent and worker agent be identified in the workflow?

File: AgentQnA/README.md [hunk 2]
```diff
@@ -5,6 +5,73 @@
...
+    direction LR
+    AG_RAG <-.-> LLM_gen
+    AG_REACT <-.-> LLM_gen
+    EM <-.-> TEI_EM
+    RET <-.-> R_RET
+    RER <-.-> TEI_RER
+
+    direction TB
+    %% Vector DB interaction
+    R_RET <-.->|d|VDB
```
Comment (xiguiw) [inline AgentQnA/README.md:69]: The 'd' on these two edges are different from other edges.
Will we keep them?

File: AgentQnA/README.md (hunk 1 above)
Comment (louie-tsai) [reply AgentQnA/README.md:39]: supervisor agent and worker agent are not strategy types of agent, so I only put strategy types in workflow.
https://github.com/opea-project/GenAIComps/tree/main/comps/agent/langchain

File: AgentQnA/README.md (hunk 2 above)
Comment (louie-tsai) [reply AgentQnA/README.md]: thanks. removed them accordingly.

**Review by xiguiw — APPROVED** [review]

**Review by NeoZhangJianyu — APPROVED** [review]

**Pull request merged**